high-level-tests/e2e/cypress/integration/pix-app/a11y.test.js

```diff
@@ -60,9 +60,9 @@ describe('a11y', () => {
       { url: '/mes-tutos/enregistres' },
       { url: '/mes-tutos/recommandes' },
       { url: '/mon-compte/informations-personnelles' },
-      { url: '/mon-compte/langue', skipFailures: true },
+      { url: '/mon-compte/langue' },
       { url: '/mon-compte/methodes-de-connexion' },
-      { url: '/plan-du-site', skipFailures: true },
+      { url: '/plan-du-site'},
```

Comment:
En réponse à la remontée d'erreur d'a11y remontée par @Jeyffrey, je propose d'ajouter une page d'erreur :

```suggestion
      { url: '/plan-du-site' },
      { url: '/assessments/fake-assessment', skipFailures: true },
```